src/lib/y2partitioner/widgets/help.rb

```diff
@@ -12,216 +12,119 @@ def included(_target)
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Parity Algorithm</b> shows the parity\nalgorithm for RAID devices with " \
-              "RAID type 5, 6 or 10."
-            )
-        when :pe_size
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>PE Size</b> shows the physical extent size\nfor LVM volume groups."
-            )
-        when :raid_version
-          # helptext for table column and overview entry
-          ret << _("<b>RAID Version</b> shows the RAID version.")
-        when :raid_type
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>RAID Type</b> shows the RAID type, also\ncalled RAID level, for RAID devices."
-            )
-        when :size
-          # helptext for table column and overview entry
-          ret << _("<b>Size</b> shows the size of the device.")
-        when :start_cyl
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Start Cylinder</b> shows the start cylinder\nof the partition."
-            )
-        when :stripes
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Stripes</b> shows the stripe number for LVM\nlogical volumes and, if greater " \
-              "than one, the stripe size  in parenthesis.\n"
-            )
-        when :type
-          # helptext for table column and overview entry
-          ret <<
-            _("<b>Type</b> gives a general overview about the\ndevice type.")
-        when :udev_id
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Device ID</b> shows the persistent device\nIDs. This field can be empty.\n"
-            )
-        when :udev_path
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Device Path</b> shows the persistent device\npath. This field can be empty."
-            )
-        when :used_by
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>Used By</b> shows if a device is used by\ne.g. RAID or LVM. " \
-              "If not, this column is empty.\n"
-            )
-        when :uuid
-          # helptext for table column and overview entry
-          ret <<
-            _(
-              "<b>UUID</b> shows the Universally Unique\nIdentifier of the file system."
-            )
-        when :vendor
-          # helptext for table column and overview entry
-          ret << _("<b>Vendor</b> shows the device vendor.")
-        else
-          raise "Unknown field #{field}"
+        text = TEXTS[field] or raise "Unknown field #{field}"
+        ret << _(text)
+        if Yast::Mode.normal
+          text2 = NORMAT_MODE_TEXTS[field]
+          ret << _(text2) if text2
```

Comment:
good point, I plan to add it to all texts, as otherwise in second one there is no gap between dot and letter.